model/src/main/java/org/projectnessie/model/Contents.java

```diff
@@ -66,11 +64,7 @@
    *
    * <p>This id is unique for the entire lifetime of this Contents object and persists across renames. Two contents with the same key
    * will have different id.
-   *
```

Comment:
What happens if we run a `>0.6.0` server w/ the `0.11.1` iceberg? Does this need to wait for an iceberg release?

Reply:
It already does not work, because the server sends the `id` field, that the Nessie client in Iceberg does not understand and fails.

Reply:
Ok raised #1310 to address more rigorously.